docs/_data/videos.yaml

```diff
@@ -3646,6 +3650,7 @@ ro-crates/bestpractices/tutorial:
   description:
   materials:
   - link: https://by-covid.github.io/gtn/galaxy-best-practices/
+  - link: https://github.com/galaxyproject/iwc/blob/main/workflows/README.md#adding-workflows
```

Comment:
just FYI, if you want to add a link text instead of displaying the full long URL, you can do something like:

```
- link: https://github.com/galaxyproject/iwc/blob/main/workflows/README.md#adding-workflows
- title: Instructions for adding workflows
```